javascript/javascript3/week2/yesNo.js

```diff
@@ -0,0 +1,180 @@
+const movies = fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+.then(response=>response.json())
+   .then(data => data.map(badmovies => badmovies).filter(movieList => movieList.rating < 6 && movieList.year >= 2000));
+   console.log(movies);
+
+ // async method 
+
+ async function asyncAwait(){
```

Comment:
It probably makes sense to rename this function to be more indicative of what it does.